packages/grafana-prometheus/src/datasource.ts

```diff
@@ -156,9 +149,7 @@ export class PrometheusDatasource
     // objects to DataSourceApi types.
     // We don't use the default processing for prometheus.
     // See standardAnnotationSupport.ts/[shouldUseMappingUI|shouldUseLegacyRunner]
-    this.annotations = {
```

Comment:
Is the comment above still true?

Reply:
The comment should be updated. I'll handle it.